### ISSUE TITLE
Health bar not working for monsters

### ISSUE THREAD

**Issue by Stefouch** [pwatson100/alienrpg#23]
The health bar appears empty because its max value is 0.

It's a minor issue, but see below:

![image](https://user-images.githubusercontent.com/47865544/101285879-fb6b0500-37e7-11eb-9fd1-70e536cfe6ac.png)

**Comment by pwatson100** [comment]
Thanks I will check into it.   I think I know what it might be.

**Comment by pwatson100** [comment]
Agghhh.  This one's gonna be a bugger.   For a character the max health is based in their strength but Creatures don't have a strength so I am not sure what I can use is the maximum??

**Comment by Stefouch** [comment]
You probably need to add a new variable in monster's sheets: current health and max health.

**Comment by pwatson100** [comment]
I have them already.  The problem is setting max health as when the sheet updates it recalculates the max health.  Also I now know why people hate coding for tokens.  I just found that if I set the health and the max health, drag the token onto the sheet when I change the health on the sheet the health/max health changes on the last token I dragged on to the scene even though the tokens are not linked.   I hate Tokens!!!
